### ISSUE TITLE
Example vuex does not work after click Android back button

### ISSUE THREAD

**Issue by Informateur** [nativescript-vue/nativescript-vue#134]
In this example, before the hard button (Android) exits, everything works. After the restart, the counter stops (data is not mutated). Tell me, how can I get around this?
https://github.com/nativescript-vue/nativescript-vue/blob/master/samples/app/app-with-vuex.js

Version without vuex works well:

```js
const Vue = require('nativescript-vue')
new Vue({
    data() { return { count: 42 }},
    template: `
    <Page>
      <StackLayout>
        <Label v-bind:text="count + ' taps left'" style="text-align: center; font-size: 30; padding: 20 0;" />
        <Button text="Tap" @tap="decrement" />
      </StackLayout>
    </Page> 
  `,
    methods: { decrement() {this.count--; } }
}).$start()
```

**Comment by Informateur** [comment]
In fact, vuex becomes read-only. It ceases to mutate. Who faced, tell me?

**Comment by Informateur** [comment]
As a workaround, I close the application by the Android button back event.
To whom it is interesting, the solution (otherwise working with nativescript vuex on Andoid turns into pain and suffering):

import * as application from "application";
import {exit} from 'nativescript-exit';

if (application.android) {
        application.on(application.exitEvent, function (args) {
            exit();
        });
    }

**Comment by rigor789** [comment]
This is probably a re-mountig / resume issue that has been fixed in `1.4.0-alpha.0` will be released as a stable release soon.
